test/functional/feature_loadblock.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test loadblock option
+
+Test the option to start a node with the option loadblock which loads
+a serialized blockchain from a file (usually called bootstrap.dat).
+To generate that file this test uses the helper scripts available
+in contrib/linearize.
+"""
+
+import os
+import urllib
+
+from test_framework.test_framework import (
+    BitcoinTestFramework,
+)
+from test_framework.util import assert_equal, wait_until
+
+
+class LoadblockTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+    def run_test(self):
+        self.nodes[1].setnetworkactive(state=False)
+        self.nodes[0].generate(100)
+
+        # Parsing the url of our node to get settings for config file
+        data_dir = self.nodes[0].datadir
+        node_url = urllib.parse.urlparse(self.nodes[0].url)
+        cfg_path = os.path.join(data_dir, "linearize.cfg")
+        hashlist_path = os.path.join(data_dir, "hashlist.txt")
+        bootstrap_file = os.path.join(self.options.tmpdir, "bootstrap.dat")
+        genesis_block = self.nodes[0].getblockhash(0)
+        block_dir = os.path.join(data_dir, "regtest", "blocks")
+
+        self.log.info("Create linearization config file")
+        with open(cfg_path, "a", encoding="utf-8") as cfg:
+            cfg.write("datadir={}\n".format(data_dir))
+            cfg.write("rpcuser={}\n".format(node_url.username))
+            cfg.write("rpcpassword={}\n".format(node_url.password))
+            cfg.write("port={}\n".format(node_url.port))
+            cfg.write("host={}\n".format(node_url.hostname))
+            cfg.write("output_file={}\n".format(bootstrap_file))
+            cfg.write("max_height=100\n")
+            cfg.write("netmagic=fabfb5da\n")
+            cfg.write("input={}\n".format(block_dir))
+            cfg.write("genesis={}\n".format(genesis_block))
+            cfg.write("hashlist={}\n".format(hashlist_path))
+
+        base_dir = self.config["environment"]["SRCDIR"]
+        linearize_dir = os.path.join(base_dir, "contrib", "linearize")
+
+        self.log.info("Run linearization of block hashes")
+        linearize_hashes_path = os.path.join(linearize_dir, "linearize-hashes.py")
+        os.system(linearize_hashes_path + " " + cfg_path + " > " + hashlist_path)
```

Comment:
Shouldn't you check the return code?

Reply:
Also, does the ` > ` work on all OS flavors? It seems that appveyor windows is passing, so it seems fine!?

Reply:
also, please use `subprocess` instead of `system` to invoke other processes from python

Reply:
> Also, does the `>` work on all OS flavors? It seems that appveyor windows is passing, so it seems fine!?

I did a brief check and did not find any issues concerning windows. I am on macOS and it's fine.

Reply:
> also, please use `subprocess` instead of `os.system` to invoke other processes from python

done

> Shouldn't you check the return code?

done